odl/solvers/advanced/proximal_operators.py

```diff
@@ -846,6 +898,61 @@ def proximal_cconj_l1(space, lam=1, g=None, isotropic=False):
     prox_factory : function
         Factory for the proximal operator to be initialized
 
+    Notes
+    -----
+    The :math:`L_1`-norm :math:`F` is the functional
+
+        .. math::
+
+            F(x) = \\lambda ||x - g||_1.
+
+    The convex conjugate :math:`F^*` of :math:`F` is given by the indicator
+    function of the set :math:`box(\\lambda)`
+
+        .. math::
+
+            F^*(y) = \\lambda I_{box(\\lambda)} \left(
+            \left| \\frac{y}{\\lambda} \\right| +
+            \left\\langle \\frac{y}{\\lambda}, g \\right\\rangle
+            \\right)
```

Comment:
I don't get this. We have a box scaled by `lambda`, and then again a scaling with `1/lambda` in the argument. That should be easy to combine, or even one of the two is redundant.